### DISCUSSION TITLE
Freud.exe Not Found

### DISCUSSION THREAD

**Discussion by DrX4000** [Estia-1a/projetGenieInfo_public#89] [Project Installation and Configuration] [answered]
Hello, 
When running the command "freud.exe -f ./images/input/image.jpeg -c dimension," I get an error (see photo). 
I'm unable to resolve it even though the file is in the correct location, and I'm not making any typos when entering the command.
![image](https://github.com/Estia-1a/projetGenieInfo_public/assets/122264304/4e14bb35-aa67-4062-976e-38c0a1d32032)

Can you help me ?

Thank you !

**Comment by CharlyBeneteau** [comment]
./build/freud.exe -f ./images/input/image.jpeg -c dimension

**Reply by dhmmasson** [reply]
It works but you should set cmake to target install

**Comment by dhmmasson** [comment, marked as answer]
If you set the target of cmake to install, it will copy freud.exe from build to the root directory. The command will then work. 
See in your readme : 4-2 Install with VS Code CMAKE extension interface, where it tells you to select ```[install]``` instead of ```[all]```
